ui/src/containers/SampleViewContainer.jsx

```diff
@@ -179,23 +178,7 @@ class SampleViewContainer extends Component {
           <Col sm={6}>
             <DefaultErrorBoundary>
               <ContextMenu
-                {...this.props.contextMenu}
                 getControlAvailability={this.getControlAvailability}
-                sampleViewActions={this.props.sampleViewActions}
```

Comment:
One fewer use of that `sampleViewActions` option, which brings us closer to removing the namespace import `import * as sampleViewActions`. :tada: